tsconfig.build.json

```diff
@@ -7,6 +7,7 @@
   "references": [
     { "path": "packages/dropUnusedDefinitions" },
     { "path": "packages/createHash" },
+    { "path": "packages/logger" },
```

Comment:
I mentioned in a top level comment that it would be nice if the directory name wasn't capitalized. Looks like it's lower-case here but the files are still capitalized in the commit.  It can be a pain to rename file case on OSX... Apparently `git mv` knows how to do that kind of rename at least. https://sebastiandedeyne.com/how-to-do-a-case-sensitive-file-rename-in-git-on-macos/

Reply:
Yeah, something about this still wasn't working, had to `git mv` to a different name entirely first, then `git mv` again. Thanks for pointing out the difficulty though, I thought I was just doing something goofy with `git mv`.